app.py

```diff
@@ -268,5 +272,67 @@ def connect_settings_db():
     return conn
 
 
-app.mount("/api", api_app)
-app.mount("/", StaticFiles(directory="static", html=True), name="Static")
+@app.get("/upload")
+def upload_file_prompt():
+    file_exists = os.path.exists(CONVERSATIONS_PATH)
+    return HTMLResponse(f"""
```

Comment:
this should go to html template

Reply:
Ok I will create a new file in the html template folder and redirect the the function to it. Is that Ok?

Reply:
in this section we will not benefit of if statement to show the delete button if the conversation json file exist or not SO I recommend to keep it 

it will be as you see if we move the file as it will show as this image and his functionality didn't work but it will be easier if this page response stay there:
![image](https://github.com/mirableio/chat-history/assets/66219679/8afa6a38-f506-47f3-b035-4331157993ed)